src/types/user.ts

```diff
@@ -0,0 +1,16 @@
+export type User = {
```

Comment:
you're not following the api specification

Reply:
If I have followed the API correctly! and 
Could you please specify which part of the API is incorrect